.gitignore

```diff
@@ -27,6 +27,7 @@ _testmain.go
 # Other dirs
 /bin/
 /pkg/
+/vendor
```

Comment:
This shouldn't be necessary since we're not vendoring in the repo.

Reply:
@calvn reverted.